### PR TITLE
ICDS Dashboard: use mapbox token from settings

### PR DESCRIPTION
**Pull request by czue**
instead of hard-coding it in the js file

### CODE REVIEW

**Review by orangejenny — APPROVED** [review]

File: custom/icds_reports/static/js/directives/awc-reports/awc-reports.directive.js [hunk 1]
```diff
@@ -2649,7 +2649,7 @@ function AwcReportsController($scope, $http, $location, $routeParams, $log, DTOp
                 url: 'https://api.mapbox.com/styles/v1/dimagi/cj2rl1t0w001f2rnr0y8hfhho/tiles/{z}/{x}/{y}?access_token={apikey}',
```
Comment (snopoke) [inline custom/icds_reports/static/js/directives/awc-reports/awc-reports.directive.js]: should this URL also be configurable? (it seems to be Dimagi specific)

**Review by snopoke — APPROVED** [review]

File: custom/icds_reports/static/js/directives/awc-reports/awc-reports.directive.js (hunk 1 above)
Comment (czue) [reply custom/icds_reports/static/js/directives/awc-reports/awc-reports.directive.js]: hmm. yeah probably, but i'd say let's address that if/when another org needs to host the dashboard.

Comment (snopoke) [reply custom/icds_reports/static/js/directives/awc-reports/awc-reports.directive.js]: we should probably do that prior to handover with govt

Comment (czue) [reply custom/icds_reports/static/js/directives/awc-reports/awc-reports.directive.js]: for sure, I imagine we'd bundle it with the key changes

Comment (czue) [reply custom/icds_reports/static/js/directives/awc-reports/awc-reports.directive.js]: added an item to https://docs.google.com/document/d/19nEiTbJbPc7IPXmGVNAePyL6WiD0i8RmT6vM11CCBsI/edit

**Pull request merged**